articles/container-apps/networking.md

```diff
@@ -192,9 +192,12 @@ There's no forced tunneling in Container Apps routes.
 
 ## Managed resources
 
-When you deploy an internal or an external environment into your own network, a new resource group prefixed with `MC_` is created in the Azure subscription where your environment is hosted. This resource group contains infrastructure components managed by the Azure Container Apps platform, and shouldn't be modified. The resource group contains Public IP addresses used specifically for outbound connectivity from your environment and a load balancer. As the load balancer is created in your subscription, there are extra costs associated with deploying the service to a custom virtual network.
+When you deploy an internal or an external environment into your own network, a new resource group prefixed with `MC_` is created in the Azure subscription where your environment is hosted. This resource group contains infrastructure components managed by the Azure Container Apps platform, and shouldn't be modified. The resource group contains Public IP addresses used specifically for outbound connectivity from your environment and a load balancer. In addition to the [Azure Container Apps billing](https://docs.microsoft.com/azure/container-apps/billing), you will be billed for the following:
+- 3 Standard Static [Public IPs](https://azure.microsoft.com/pricing/details/ip-addresses/) if using an internal environment, or 4 Standard Static [Public IPs](https://azure.microsoft.com/pricing/details/ip-addresses/) if using an external environment,
+- 2 Standard [Load Balancers](https://azure.microsoft.com/pricing/details/load-balancer/) if using an internal environment, or 1 Standard [Load Balancer](https://azure.microsoft.com/pricing/details/load-balancer/) if using an external environment, each with less than 6 rules, plus the cost of Data processed (GB). The Data processed (GB) includes both ingress and egress for management operations.
```

Comment:
```suggestion
- Three standard static [public IPs](https://azure.microsoft.com/pricing/details/ip-addresses/) if using an internal environment, or four standard static [public IPs](https://azure.microsoft.com/pricing/details/ip-addresses/) if using an external environment.
- Two standard [Load Balancers](https://azure.microsoft.com/pricing/details/load-balancer/) if using an internal environment, or one standard [Load Balancer](https://azure.microsoft.com/pricing/details/load-balancer/) if using an external environment, each with less than six rules, plus the cost of data processed (GB). The data processed (GB) includes both ingress and egress for management operations.
```

This is a bit of a run on sentence: 

> Two standard [Load Balancers](https://azure.microsoft.com/pricing/details/load-balancer/) if using an internal environment, or one standard [Load Balancer](https://azure.microsoft.com/pricing/details/load-balancer/) if using an external environment, each with less than six rules, plus the cost of data processed (GB)

but I wasn't sure how to fix it up without changing the meaning. Can you rephrase a bit, please?

Reply:
Two standard [Load Balancers](https://azure.microsoft.com/pricing/details/load-balancer/) if using an internal environment, or one standard [Load Balancer](https://azure.microsoft.com/pricing/details/load-balancer/) if using an external environment. Each load balancer has less than six rules. The cost of data processed (GB) includes both ingress and egress for management operations.